packages/uui-color-swatches/lib/uui-color-swatches.test.ts

```diff
@@ -0,0 +1,22 @@
+import { html, fixture, expect } from '@open-wc/testing';
+import { UUIColorSwatchesElement } from './uui-color-swatches.element';
+
+import '@umbraco-ui/uui-color-swatch/lib';
+
+describe('UUIColorSwatchesElement', () => {
+  let element: UUIColorSwatchesElement;
+
+  beforeEach(async () => {
+    element = await fixture(
+      html` <uui-color-swatches label="Color swatches"></uui-color-swatches> `
```

Comment:
I added it. I think it makes sense for that to have a label. I think I will also add a role of `radio-group` to it and set the role to `radio` on slotted swatches in `uui-color-swatches`,  since that is what it actually is accesibility wise